charts/redpanda/templates/statefulset.yaml

```diff
@@ -73,10 +73,9 @@ spec:
         - name: tuning
           image: {{ .Values.image.repository }}:{{ template "redpanda.tag" . }}
           command:
-            - bash
+            - /bin/bash
```

Comment:
```suggestion
            - rpk redpanda tune all
```

Reply:
it did not work, the container would not start.

Reply:
I can not believe that previously it wouldn't start. At least one test should catch that.